src/js/components/Ballot/PositionRowListCompressed.jsx

```diff
@@ -140,23 +208,40 @@ class PositionRowListCompressed extends Component {
     this.onCachedPositionsChange();
   }
 
-  onClickShowOrganizationModalWithPositions () {
+  getTalkingAboutText = () => {
     const { ballotItemWeVoteId } = this.props;
-    // console.log(ballotItemWeVoteId)
-    // console.log('onClickShowOrganizationModalWithPositions, ballotItemWeVoteId:', ballotItemWeVoteId);
-    AppObservableStore.setOrganizationModalBallotItemWeVoteId(ballotItemWeVoteId);
-    AppObservableStore.setShowOrganizationModal(true);
-    AppObservableStore.setHideOrganizationModalBallotItemInfo(true);
-  }
+    let itemName = '';
 
-  onClickShowOrganizationModalWithBallotItemInfoAndPositions () {
-    const { ballotItemWeVoteId } = this.props;
-    AppObservableStore.setOrganizationModalBallotItemWeVoteId(ballotItemWeVoteId);
-    AppObservableStore.setShowOrganizationModal(true);
-  }
+    if (ballotItemWeVoteId.includes('cand')) {
+      const candidate = CandidateStore.getCandidateByWeVoteId(ballotItemWeVoteId);
+      itemName = candidate ? candidate.ballot_item_display_name : '';
+    } else if (ballotItemWeVoteId.includes('meas')) {
+      itemName = MeasureStore.getMeasureName(ballotItemWeVoteId);
+    }
+
+    const endorsementCount = this.getEndorsementCount();
+    if (endorsementCount === 0) return '';
+    if (endorsementCount === 1) return `is talking about ${itemName}`;
+    return `are talking about ${itemName}`;
+  };
+
+  getOrganizationCount = () => {
+    const { positionListFromFriendsHasBeenRetrieved } = this.props;
+    if (!positionListFromFriendsHasBeenRetrieved) return 0;
+
+    const uniqueOrgs = new Set(
+      positionListFromFriendsHasBeenRetrieved.map((position) => position.speaker_we_vote_id),
+    );
+    return uniqueOrgs.size;
```

Comment:
I think this needs to be length instead of size. This won't crash now (because positionListFromFriendsHasBeenRetrieved is always empty), but please test/update in your next pull request.